src/components/Home/Home.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Home() {
+  return (
+    <div />
+  );
+}
+
+export default Home;
```

Comment:
- **[OPTIONAL]** Is there any need for this component? I think we can remove it since it only imported into `src/components/index.js` file which isn't used as well.